pandas/core/generic.py

```diff
@@ -5909,6 +5909,97 @@ def infer_objects(self: FrameOrSeries) -> FrameOrSeries:
             )
         ).__finalize__(self)
 
+    def as_nullable_dtypes(
+        self: FrameOrSeries, convert_integer: bool_t = True
+    ) -> FrameOrSeries:
+        """
+        Convert columns of DataFrame or a Series to types supporting ``pd.NA``.
+
+        If the dtype is ``object``, if possible, convert to ``StringDtype``,
```

Comment:
This content may fit better in a "Notes" section since it's explaining the details of the conversion.

Reply:
OK, moved.